python3/raygun4py/raygunmsgs.py

```diff
@@ -191,6 +193,60 @@ def __init__(self, exc_type=None, exc_value=None, exc_traceback=None, options=No
                         if 'localVariables' in frame:
                             frame['localVariables'] = None
 
+    def _check_and_modify_payload_size(self, options, max_size_kb=128):
```

Comment:
Convention thing - do we still want to prefix this with an underscore given that it is now being used external to this class?

Reply:
Good point, will change that now